packages/ppx_deriving_cmdliner/ppx_deriving_cmdliner.0.5.1/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "Seb Mondet <seb@mondet.org>"
+authors: [
+  "Isaac Hodes <isaachodes@gmail.com>"
+  "B. Arman Aksoy <arman@aksoy.org>"
+  "Seb Mondet <seb@mondet.org>"
+  "Nick Zalutskiy <nick@const.fun>"
+  "Armaël Guéneau <armael.gueneau@ens-lyon.fr>"
+]
+homepage: "https://github.com/hammerlab/ppx_deriving_cmdliner"
+bug-reports: "https://github.com/hammerlab/ppx_deriving_cmdliner/issues"
+dev-repo: "git+https://github.com/hammerlab/ppx_deriving_cmdliner.git"
+doc: "http://hammerlab.github.io/ppx_deriving_cmdliner"
+license: "MIT"
+version: "0.5.1"
+tags: ["syntax" "cli"]
+build: [
+  ["jbuilder" "subst" "-n" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+run-test: [
+  ["jbuilder" "runtest" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03"}
+  "cmdliner" {>= "1.0.0"}
+  "result"
+  "ppx_deriving" {>= "4.0" & < "5.0"}
+  "dune" {< "2.0"}
```

Comment:
Is this expected? Looking at the code it seems to use dune, not jbuilder as the build/test step imply.

Reply:
I fixed it directly. If it doesn't work, our CI will tell us.

Reply:
Thank you @kit-ty-kate. Yes, I should have updated the project opam here and upstream. My next contribution upstream will be moving to dune 2.0+, so I will modify the opam template in the upstream repo then. I do not believe there are any incompatible features from dune 1.0 in use, so it should be straightforward.

Reply:
Looks like the x86_64 Linux test failed to start. It errored out without a log after 1 second. Others seem to have worked fine.